applications/search/src/pages/search-page/search-page.jsx

```diff
@@ -243,6 +243,19 @@ export const SearchPage = props => {
             />
           </Route>
         </Switch>
+        <div className="twitter-container">
+          <div className="twitter">
```

Comment:
Her kan du også bruke bootstrap sine klasser for margin og padding, du unngår da pixel, men får bruker rem/em som justeres i forhold til skrifstørrelse ved f.eks. zoom eller browser-font. Her kan det bli noe som: 'p-5  mb-5 fdk-bg-white '